documentation/asciidoc/computers/config_txt/audio.adoc

```diff
@@ -2,6 +2,12 @@
 
 The onboard audio output uses config options to change the way the analogue audio is driven, and whether some firmware features are enabled or not.
 
+=== `audio_pwm_mode`
+
+`audio_pwm_mode=1` selects legacy low-quality analogue audio from the 3.5mm AV jack.
+
+`audio_pwm_mode=2` (the default) selects high quality analogue audio using an advanced modulation scheme. Note that this option uses more GPU compute resources and can interfere with some use cases.
```

Comment:
Just a stylistic point, rather than a content thing. If you're adding an aside, out of the main text, the standard way to do so should be;

```
[NOTE]
====
This option uses more GPU compute resources and can interfere with some use cases.
====
```